src/client/firefox/commands.js

```diff
@@ -305,6 +305,10 @@ async function setPausePoints(sourceId: SourceId, pausePoints: PausePoints) {
   return sendPacket({ to: sourceId, type: "setPausePoints", pausePoints });
 }
 
+async function setSkipPausing(shouldSkip: boolean) {
+  return sendPacket({ skip: shouldSkip, to: "store", type: "source" });
```

Comment:
This call results in Firefox receiving a request (yay!) but these keys don't appear to be right because the `actor` on the Firefox side is `null`;  hopefully figure this out tomorrow.